### PR TITLE
[OSDEV-885] Implement reset database to original state checkbox

### PR DESCRIPTION
**Pull request by killev**
[OSDEV-885](https://opensupplyhub.atlassian.net/browse/OSDEV-885) - Implement option to reset database for `Dev`, `Test` and `Pre-prod` environmet to `Deploy to AWS` pipleine

### CODE REVIEW

File: src/dedupe-hub/api/app/main.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ async def initialize():
         if end_offset == 0:
             log.warning(f'Topic ({settings.topic_dedupe_basic_name}) has no messages (log_end_offset: '
                         f'{end_offset}), skipping initialization ...')
-            return
+            return True
```
Comment (roman-stolar) [inline src/dedupe-hub/api/app/main.py:102]: Why we need to return True here?

Comment (killev) [reply src/dedupe-hub/api/app/main.py:102]: Because in the end we had a conditions:

**Review by roman-stolar — APPROVED** [review]
LGTM

**Review by vladsha-dev — APPROVED** [review]
LGTM

**Pull request merged**